test/helpers/loans_helper_test.rb

```diff
@@ -8,9 +8,11 @@ class LoansHelperTest < ActionView::TestCase
   end
 
   test "humanize_due_date returns tomorrow when due date is tomorrow" do
-    @loan = build(:loan, due_at: 1.days.from_now)
-
-    assert_equal "tomorrow", humanize_due_date(@loan)
+    Timecop.freeze(Date.new(2020, 1, 1)) do
```

Comment:
Can we use the built-in `travel_to` method from Rails instead of timecop here?

Reply:
I removed the timecop.
I also found that the root cause is the `from_now` method which use Time.current as input.
Time.current return UTC time that's why it has timezone problem
https://apidock.com/rails/ActiveSupport/Duration/from_now.

I changed to use Date.today + 1.day, which always uses local time, so we don't need change Time.now anymore.